src/Cosmology.jl

```diff
@@ -148,22 +148,37 @@ hubble_time(c::AbstractCosmology, z) = hubble_time0(c)/E(c,z)
 
 Z(c::AbstractCosmology, z::Real; kws...) =
     QuadGK.quadgk(a->1/a2E(c,a), scale_factor(z), 1; kws...)[1]
+Z(c::AbstractCosmology, z₁::Real, z₂::Real; kws...) =
```

Comment:
I think you can replace the method above with the new one you defined and set `z₂::Real=0` as default value